### ISSUE TITLE
[DATA] CVE-2017-1000410

### ISSUE THREAD

**Issue by gotthardp** [nluedtke/linux_kernel_cves#186]
**Change Type Requested**
Update

**CVE id number**
CVE-2017-1000410

**References**
https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=linux-4.14.y&id=02462928e2234eea95dd1ce05ca1f2c02088dc13

**Additional context**
The references to fixes in the individual branches are wrong-- they refer to fixes for CVE-2017-1000251.

For example, in the 4.14 branch, the CVE-2017-1000410 fix is in 02462928e2234eea95dd1ce05ca1f2c02088dc13, see the referece above. The currently referenced e860d2c904d1a9f38a24eb44c9f34b8f915a6ea3 is a fix for CVE-2017-1000251.

**Comment by gotthardp** [comment]
I commit ids seem still wrong. For example, in 4.14_security.txt:
```
CVEs fixed in 4.14-rc1:
  CVE-2017-1000410: e860d2c904d1a9f38a24eb44c9f34b8f915a6ea3 Bluetooth: Prevent stack info leak from the EFS element.
```
The commit string (the text) is ok, but the commit hash is still the same as before, i.e. wrong.
At least the commit string does not match the hash since the
hash e860d2c904d1a9f38a24eb44c9f34b8f915a6ea3 = "Bluetooth: Properly check L2CAP config option output buffer length". 
It's the hash 02462928e2234eea95dd1ce05ca1f2c02088dc13 that is "Bluetooth: Prevent stack info leak from the EFS element."

**Comment by nluedtke** [comment]
This is probably due to the cache not being fully deleted before the text file was regenerated. This will be in tomorrow's push
